src/app/service/poll-item.service.ts

```diff
@@ -35,8 +35,17 @@ export class PollItemService {
    * @param pollItem Poll item
    */
   create<T extends MultipleChoiceItem | QuizItem | OpenTextItem>(pollItem: T): Observable<T> {
-    const endpointFraction = this.tools.convertCamelCaseToKebabCase(pollItem.constructor.name)
```

Comment:
Where does `this.tools.convertCamelCaseToKebabCase()` come from? Still, better approach right now instead of having to work with reflections... xD

Reply:
`this.tools.convertCamelCaseToKebabCase()` comes from our tools service. I did not delete it because it could be helpful in the future.